packages/localization/src/DateUtils.js

```diff
@@ -25,4 +25,16 @@ DateUtils.diffInDays = (higherDate: Date, lowerDate: Date) => {
   return Math.ceil(diffMs / (1000 * 3600 * 24));
 };
 
+/**
+ *
+ * Returns whether firstDate and secondDate are the same day in local time
+ *
+ */
```

Comment:
This comment is not only quite big but probably not necessary? 😃

Reply:
Agreed, I had a more complicated function before, so that's why I had thought about describing it. I will remove it :smile: